nss-day-cohort-43-csharp/MVC-Green-Monkeys
# Tags

# Description
User can see the full list of tags and add a new tag.
Fixes # 13 & 14
## Type of change
# Testing Instructions
When logged on, you should be able to navigate to the Tags tab and see full list of tags and add a new tag.
# Checklist:
